docs/source/reference/visualization/index.rst

```diff
@@ -8,7 +8,7 @@ The :mod:`~optuna.visualization` module provides utility functions for plotting
 .. note::
     In the :mod:`optuna.visualization` module, the following functions use plotly to create figures, but `JupyterLab`_ cannot
     render them by default. Please follow this `installation guide`_ to show figures in
-    `JupyterLab`_.
+    `JupyterLab`_. The `plot_param_importances` requires the pip package `scikit-learn`.
```

Comment:
This note block originally focused on the use of the visualization functions in `JupyterLab`, but the `scikit-learn` package is required even if users do not use `JupyterLab`. 

<img width="599" alt="image" src="https://user-images.githubusercontent.com/3255979/155635320-d87638bf-8431-49bb-ac2c-bbd89edb335b.png">

How about creating a new note block, for example, as follows?

```suggestion
    `JupyterLab`_. 
.. note::
    The :func:`~optuna.visualization.plot_param_importances` requires the Python package of `scikit-learn <https://github.com/scikit-learn/scikit-learn>`_.
```

Reply:
@ll7 If you are busy, I'll commit the suggestion to your branch. @himkt will review the change in that case, so please feel free to ask us.